pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -277,6 +277,19 @@ func (ds *dockerService) StopPodSandbox(ctx context.Context, r *runtimeapi.StopP
 	}
 
 	// TODO: Stop all running containers in the sandbox.
+	go func() {
```

Comment:
is this type of async stop safe?

Reply:
I use async because of stop all containers are not ``StopPodSandbox`` main job